python/social_bot/worlds/grocery_ground.world

```diff
@@ -9,11 +9,18 @@
       <max_step_size>0.001</max_step_size>
       <real_time_factor>1</real_time_factor>
       <real_time_update_rate>0</real_time_update_rate>
+      <max_contacts>20</max_contacts>
       <ode>
-        <solver>quick</solver>
+        <solver>
+          <type>quick</type>
+          <iters>50</iters>
+          <sor>1.3</sor>
+        </solver>
         <constraints>
           <contact_max_correcting_vel>1</contact_max_correcting_vel>
-          <contact_surface_layer>0.001</contact_surface_layer>
+          <contact_surface_layer>0.002</contact_surface_layer>
+          <cfm>0.0001</cfm>
+          <erp>0.3</erp>
```

Comment:
explain briefly what these do?  Maybe include a link to some external resource or documentation?

Reply:
this actually make the joint 'soft', thus increasing the numerical stability. check the document here:
http://ode.org/ode-latest-userguide.html#sec_3_7_0